python/paddle/tensor/creation.py

```diff
@@ -54,6 +55,166 @@
 ]
 
 
+@dygraph_only
+def to_tensor(data, dtype=None, stop_gradient=True, pin_memory=False):
+    """
+    :api_attr: imperative
+
+    :alias_main: paddle.to_tensor
+	:alias: paddle.tensor.to_tensor, paddle.tensor.creation.to_tensor
```

Comment:
Is the indentation correct here? and will these marks be displayed in preview document?

Reply:
Done